README.md

```diff
@@ -0,0 +1,35 @@
+# Mouse Recorder
+###A cheap, modular, behavior recording program
+![](http://s16.postimg.org/yek83sdv9/Mouserecord_logo.jpg 2)
+
+# Purpose
+
+The program's is designed to record rodent behavior at specific intervals of time, based on the initiation of a trigger event (e.g., the press of a lever.) Moreover, the program operates through Raspberry Pi-based system to significantly reduce cost as opposed to conventional recording apparatus.
```

Comment:
"Moreover, the program operates through Raspberry Pi-based system to significantly reduce cost as opposed to conventional recording apparatus."

This line reads funny. I think it could be rewritten.